CHANGELOG.md

```diff
@@ -1,5 +1,165 @@
+- Gradle: Merge CodegenPluginExtension inside ReactAppExtension ([022f188750](https://github.com/facebook/react-native/commit/022f1887509fb2206441a0735cae91a6e2ddba04) by [@cortinico](https://github.com/cortinico))
+- Use new Locale API on Android 11 (API 30)+ ([b7c023a8c1](https://github.com/facebook/react-native/commit/b7c023a8c1122500c6ceb7de2547569b3b9251ba))
+- Hermes initialization will no longer need an explicit configuration. ([a40f973f58](https://github.com/facebook/react-native/commit/a40f973f58609ca717fac63bc501d5cf93b748ad) by [@Ashoat](https://github.com/Ashoat))
+- Bumped Gradle to 7.1.1 ([b1120c6a65](https://github.com/facebook/react-native/commit/b1120c6a6513ec24568abc6bbe60dd57ef1dec96) by [@cortinico](https://github.com/cortinico))
+- Bumped Kotlin to 1.4.32 ([6a2b6c2f3e](https://github.com/facebook/react-native/commit/6a2b6c2f3ef63cf0ff733daf75005ee32eebdd00) by [@cortinico](https://github.com/cortinico))
+- Bumped Gradle project version to 7.0.2 ([85249cafe8](https://github.com/facebook/react-native/commit/85249cafe8870ab8f47c38569b106555ce2f8527) by [@cortinico](https://github.com/cortinico))
+- Remove `"high"` and `"balanced"` as values for `android_hyphenationFrequency` on `Text` ([a0d30b848a](https://github.com/facebook/react-native/commit/a0d30b848a07480d0fccec608a62a505c71f8cac))
+
+#### iOS specific
+
+- Setup a new macro to gate the core packager functionality ([a8cd8f7696](https://github.com/facebook/react-native/commit/a8cd8f769695621e4624a14d1da7f355c4544cbe) by [@jimmy623](https://github.com/jimmy623))
+- Fixed inaccurate comment ([c744528da1](https://github.com/facebook/react-native/commit/c744528da1d6094f73b087d0ef90e33a9fa105ed) by [@philIip](https://github.com/philIip))
+- Remove iOS 11 version check ([398595e074](https://github.com/facebook/react-native/commit/398595e07483fa8f45579de4ca1aee9585e20620) by [@ken0nek](https://github.com/ken0nek))
+- Optimized font handling ([4ac42d88ef](https://github.com/facebook/react-native/commit/4ac42d88ef60ae3fed7319851d47b93e98ac9afa) by [@Adlai-Holler](https://github.com/Adlai-Holler))
+
+### Deprecated
+
+#### Android specific
+
+#### iOS specific
+
+### Removed
+
+- Removed unnecessary global variable `GLOBAL`. ([a101fc768c](https://github.com/facebook/react-native/commit/a101fc768cedc7ac9754006e5b7292bb7084ab54) by [@rubennorte](https://github.com/rubennorte))
+- Chore: removed unused files ([64aa1e5ffe](https://github.com/facebook/react-native/commit/64aa1e5ffe5d577c04cabb3692246b956f65597b) by [@ecreeth](https://github.com/ecreeth))
+
+#### Android specific
+
+- Remove DatePickerAndroid from react-native-github ([7a770526c6](https://github.com/facebook/react-native/commit/7a770526c626e6659a12939f8c61057a688aa623) by [@andresantonioriveros](https://github.com/andresantonioriveros))
```

Comment:
```suggestion
- Remove DatePickerAndroid from react-native-github ([7a770526c6](https://github.com/facebook/react-native/commit/7a770526c626e6659a12939f8c61057a688aa623) by [@andresantonioriveros](https://github.com/andresantonioriveros))
```

This is breaking change, since the whole component is being removed.